docs/xks/developer-guide/monitoring.md

```diff
@@ -0,0 +1,63 @@
+---
+id: monitoring
+title: Monitoring
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Monitoring applications is an especially important feature when developing microservices. Currently the only supported monitoring solution XKS
+supports is [Datadog](https://www.datadoghq.com), but more can be added in the future.
+
+## Datadog
+
+When Datadog monitoring is used the [Datadog Operator](https://github.com/DataDog/datadog-operator) will be added to the cluster. Ontop of deploying a
+Datadog agent to every node to collect, metrics, logs, and traces it also adds the ability to create Datadog monitors from the cluster. The Datadog
+agent handles all communication with the Datadog API meaning that individual applications do not have to deal with thing such as authentication.
+
+### Logging
+
+All logs written to `stdout` from applications in the tenant namespace will be collected by the Datadog agents. This means that no additional
+configuration has to be done to the application, other than making sure the logs are written to the correct destination.
```

Comment:
What about stderr? Logs should normally be written to stderr.

Reply:
Perhaps it would be worth saying something like "This means that `kubectl logs` and Datadog will display the same information"?